WindowsMRAssetConverter/README.md

```diff
@@ -9,42 +9,66 @@ WindowsMRAssetConverter _&lt;path to GLTF/GLB&gt;_
+  - `holographic`: optimizes assets for HoloLens using the [MSFT_packing_normalRoughnessMetallic](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Vendor/MSFT_packing_normalRoughnessMetallic) extension.
+  - `all`: creates assets optimized for both holographic and desktop devices, but with a larger file size.
+
+- `-compatibility-mode <on | off>`
+  - **Default:** `on`
+  - Generates assets that are also compatible with versions of Windows 10 older than the Spring Creators Update (version 1803).
+  - Enabling compatibility mode results in assets with a larger file size. If your app is compatible with Windows 10 1803+ only, it is recommended to set `-compatibility-mode off`.
+  - This setting does not have any effect on the Holographic platform.
+
 - `-lod <path to each lower LOD asset in descending order of quality>`
   - Specifies a list of assets that represent levels of detail, from higher to lower, that should be merged with the main asset and used as alternates when the asset is displayed from a distance (with limited screen coverage).
 
 - `-screen-coverage <LOD screen coverage values>`
-  - Specifies the maximum screen coverage values for each of the levels of detail, according to the [MSFT_lod](https://github.com/sbtron/glTF/tree/MSFT_lod/extensions/Vendor/MSFT_lod) extension specification.
+  - Specifies the maximum screen coverage values for each of the levels of detail, according to the [MSFT_lod](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Vendor/MSFT_lod) extension specification.
+
+- `-share-materials`
+  - If enabled, creates assets that share materials between different levels of detail. 
+  - This assumes all LOD documents use the same indices for each material, and uses the textures from the most detailed level.
 
-- `-temp-directory <temporary folder, default is the system temp folder for the user>`
+- `-temp-directory <temporary folder>`
+  - **Default:** system temp folder for the user
   - Allows overriding the temporary folder where intermediate files (packed/compressed textures, converted GLBs) will be placed.
 
-- `-max-texture-size <Max texture size in pixels, default is 512>`
+- `-max-texture-size <Max texture size in pixels>`
+  - **Default:** 512
   - Allows overriding the maximum texture dimension (width/height) when compressing textures. The recommended maximum dimension in the [documentation](https://developer.microsoft.com/en-us/windows/mixed-reality/creating_3d_models_for_use_in_the_windows_mixed_reality_home#texture_resolutions_and_workflow) is 512, and the allowed maximum is 4096.
 
+- `-replace-textures`
+  - If enabled, replaces all textures with their DDS compressed equivalents during the compression step. 
+  - This results in a smaller file size, but the resulting file will not be compatible with most glTF viewers.

```

Comment:
This also sounds sort of like 'compatibility-mode', as in one might expect that when 'compatibility-mode' is off this flag would get applied automatically.

Reply:
Makes sense. I think I'll get rid of -compatibility-mode entirely given this comment and @najadojo 's comments above.